lib/src/Uncertainty/Algorithm/Transformation/NatafEllipticalCopula/NatafEllipticalCopulaEvaluation.cxx

```diff
@@ -112,29 +111,32 @@ Matrix NatafEllipticalCopulaEvaluation::parameterGradient(const Point & ) const
 /* Accessor for input point dimension */
 UnsignedInteger NatafEllipticalCopulaEvaluation::getInputDimension() const
 {
-  return cholesky_.getDimension();
+  return inverseCholesky_.getDimension();
 }
 
 /* Accessor for output point dimension */
 UnsignedInteger NatafEllipticalCopulaEvaluation::getOutputDimension() const
 {
-  return cholesky_.getDimension();
+  return inverseCholesky_.getDimension();
 }
 
 /* Method save() stores the object through the StorageManager */
 void NatafEllipticalCopulaEvaluation::save(Advocate & adv) const
 {
   EvaluationImplementation::save(adv);
   adv.saveAttribute( "standardDistribution_", standardDistribution_ );
-  adv.saveAttribute( "cholesky_", cholesky_ );
+  adv.saveAttribute( "inverseCholesky_", inverseCholesky_ );
 }
 
 /* Method load() reloads the object from the StorageManager */
 void NatafEllipticalCopulaEvaluation::load(Advocate & adv)
 {
   EvaluationImplementation::load(adv);
   adv.loadAttribute( "standardDistribution_", standardDistribution_ );
-  adv.loadAttribute( "cholesky_", cholesky_ );
+  if (adv.hasAttribute("inverseCholesky_"))
+    adv.loadAttribute("inverseCholesky_", inverseCholesky_);
+  else
+    adv.loadAttribute("cholesky_", inverseCholesky_); // OT < 1.24
```

Comment:
Shouldn't we inverse the cholesky factor stored in the old versions?

Reply:
only the attribute name was wrong: it was the inverse all along!
see NormalCopula::getIsoProbabilisticTransformation